docs/use/markdown.md

```diff
@@ -103,6 +103,22 @@ b = "Python code!"
 print(f"{a} {b}")
 ```
 
+**Code from Files:**
+
+`myst_nb` provides a convenience feature for importing executable code into a `{code-cell}`
+from a file. This can be useful when you want to share code between documents. To do this
+you specify a `file` metadata attribute such as:
+
+````md
+```{code-cell} ipython3
+:file: <path>
```

Comment:
```suggestion
:myst_file: <path>
```

Reply:
@chrisjsewell not sure why you want this to be named `myst_file` as they are files that contain code such as `py` and `jl` files.

Reply:
I updated the error name from `MystFileParsingErorr` to `CodeFileParsingError` to avoid confusion.

Reply:
my tentativeness here is that `file` is a pretty generic key and on the off-chance anyone already has this key in their metadata it will now break their builds. Is there perhaps something a little less generic; `import_file` or whatever?